src/steps.js

```diff
@@ -166,7 +166,7 @@ module.exports.configureAwsSdk = params => {
     region: params.region,
   };
   aws.config.update(awsOptions);
-  aws.config.s3 = { params: { Bucket: params.bucket } };
+  aws.config.s3 = { params: { Bucket: params.bucket , Prefix: params.prefix } };
```

Comment:
There is an excess space here, I think

Also, if we set `Prefix` to the whole S3 object, do we need to add prefix to all the file names? Could you add a doc link on the behavior of the `Prefix` parameter here? I struggle to find it.

Reply:
I'm not sure, I think (not sure, i'll have to check) Prefix parameter in params object are not used by every command of s3. I think it's used on some download file methods but now on upload ones.